subprojects/jacoco/src/main/java/org/gradle/testing/jacoco/tasks/JacocoReportBase.java

```diff
@@ -48,6 +51,7 @@
 @Incubating
 public abstract class JacocoReportBase extends JacocoBase {
 
+    private Set<JacocoTaskExtension> taskExtensions = new HashSet<JacocoTaskExtension>();
```

Comment:
Can you point me to that specific documentation? If that's the case then I agree that we shouldn't go ahead with the change.